CHANGELOG.md

```diff
@@ -10,6 +10,49 @@ To see unreleased changes, please see the [CHANGELOG on the main branch guide](h
 
 <!-- towncrier release notes start -->
 
+## [0.19.0] - 2023-05-28
+
+### Packaging
+
+- Correct dependency on syn to version 1.0.85 instead of the incorrect version 1.0.56. [#3152](https://github.com/PyO3/pyo3/pull/3152)
+
+### Added
+
+- Support `text_signature` option (and automatically generate signature) for `#[new]` in `#[pymethods]`. [#2980](https://github.com/PyO3/pyo3/pull/2980)
+- Added support for converting to and from Python's `decimal.Decimal` and `rust_decimal::Decimal`. [#3016](https://github.com/PyO3/pyo3/pull/3016)
```

Comment:
I now it is busy work that we should have done when merging these PR, but maybe we can harmonize the tense to present? It hopefully is less work to do it once while preparing the changelog.

Reply:
Agreed, I'll fixup when pulling in #3188. I was so tired last night in the end 😅